app/models/triannon/annotation_ldp.rb

```diff
@@ -10,43 +10,44 @@ def graph
 
     # RDF::Graph without any back end (e.g. LDP, Fedora) triples
     def stripped_graph
-      RDF::LDP.remove_ldp_triples (RDF::FCRepo4.remove_fedora_triples(graph))
+      OA::Graph.remove_ldp_triples(OA::Graph.remove_fedora_triples(graph))
     end
 
     def base_uri
-      res = graph.query Triannon::Graph.anno_query
+      res = graph.query OA::Graph.anno_query
       res.first.s
     end
 
     # @return [Array<String>] the uris of each LDP body resource
     def body_uris
-      q = Triannon::Graph.anno_query.dup
-      q << [:s, RDF::OpenAnnotation.hasBody, :body_uri]
+      q = OA::Graph.anno_query.dup
+      q << [:s, RDF::Vocab::OA.hasBody, :body_uri]
       solns = graph.query q
       result = []
-      solns.distinct.each { |soln| 
-        result << soln.body_uri 
+      solns.distinct.each { |soln|
+        result << soln.body_uri
       }
       result
```

Comment:
Yes, this is not a blocker on this PR, as it's just a matter of style and/or performance rather than an issue of code behavior.